opa-iptables/docs/opa-iptables-example/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+)
+
+var (
+	port = ""
+)
+
+func getEnv(env string) string {
+	e := os.Getenv(env)
+	return e
+}
+
+func main() {
+	port = getEnv("PORT")
+	if port == "" {
+		port = "8080"
```

Comment:
I would not use such a popular port since there are chances of collision and you do handle ListenandServe() errors